BLSQ/iaso
# IA-2881: Completeness should be limited to the user assigned geography

Explain what problem this PR is resolving
- Completeness should be limited to the user assigned geography
- The PR is about to fix it
Related JIRA tickets : [IA-2881](https://bluesquare.atlassian.net/browse/IA-2881)

## Self proofreading checklist


## Changes
- Added a filter on org_unit_path_descendants if the user has assigned org units

## How to test
- Create or edit a user and assign org units to him
- Check on completeness data if only submissions related to the org units(assigned to the user) and its descendants are the only being displayed

## Print screen / video
[Screencast from 2024-06-03 15-35-45.webm](https://github.com/BLSQ/iaso/assets/19631540/5b8283c1-1eb0-42fc-a8c1-c11a0908e438)
[Screencast from 2024-06-03 15-36-40.webm](https://github.com/BLSQ/iaso/assets/19631540/e037361e-e793-45b0-abc8-f73374a8ce94)




[IA-2881]: https://bluesquare.atlassian.net/browse/IA-2881?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
